src/content/docs/accounts/accounts-billing/account-setup/downgradecancel-account.mdx

```diff
@@ -57,21 +57,49 @@ If you have a Pro or Enterprise edition with [a long-term contract](/docs/accoun
   </Collapser>
 </CollapserGroup>
 
-## Delete organization or accounts [#cancel]
+## Cancel an account [#cancel-account]
 
-If you're seeking to delete one or more accounts in an organization, but keep your organization, contact [support](/docs/using-new-relic/welcome-new-relic/get-started/find-help-use-support-portal).
+If you're using the Pro or Enterprise edition, you can cancel one or more accounts while keeping your organization.
 
-If you're seeking to delete your New Relic organization entirely, including any and all accounts within it, your organization must meet the following requirements:
+When you cancel an account:
 
-* It has only one account, and one user.
-  * If you have more than one account in your organization, contact [support](/docs/using-new-relic/welcome-new-relic/get-started/find-help-use-support-portal).
+* It is disabled, but it's still present in your organization in **Accounts > Cancelled accounts**.
+* Users, groups, and custom roles remain, but access grants that are assigned to that cancelled account lose all connections to the account. You should also remove any access grant associated with this account, otherwise users may see errors when they try to view it.
+* Any API keys associated with a deleted account become inactive and can't be reassigned to another account. 
+
+### Who can cancel an account [#who-cancel]
+
+To cancel an account you must be a user who is:
+
+* In an organization that is using the Pro or Enterprise edition.
+* A full platform user.
+* In a group that has the Organization [administrative setting](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts/#admin-settings).
+
+### How to cancel an account [#how-to-cancel]
+
+You can only cancel an account using our NerdGraph API&mdash;not through the UI.
+
+Make sure you're a user who can cancel an account, and then see the NerdGraph tutorial [Cancel an account](/docs/apis/nerdgraph/examples/manage-accounts-nerdgraph).
+
+## Delete your organization [#cancel]
+
+If you want to delete your New Relic organization entirely, you can only do so if your organization has downgraded to free and has just one remaining account. 
+
+### When you can delete an organization [#when-delete-org]
+
+Before trying to delete your organization, please confirm the folloiwng: 
+
+* Your organization only has one account and one user.
+  * If you have more than one account in your organization, either use the [UI](#delete-account) to delete all but one of your accounts, or contact [support](/docs/using-new-relic/welcome-new-relic/get-started/find-help-use-support-portal) for help with this.
```

Comment:
This actually would need a revise. If an org has more than one account, they won't be able to self-serve, they'll need to reach support. They can certainly cancel any additional accounts using Nerdgraph, but the account will still remain in the org under the "cancelled accounts" tab. We could update this to say:

"If you have more than one account in your organization, first use our Nerdgraph API to cancel any additional accounts (see here; this can just point back to the above section). Reach out to new relic support to remove any cancelled accounts."